front/lib/api/assistant/agent_usage.ts

```diff
@@ -171,36 +172,42 @@ export async function getAgentUsage(
 export async function agentMentionsUserCount(
   workspaceId: number
 ): Promise<mentionCount[]> {
-  // We retrieve mentions from conversations in order to optimize the query
-  // Since we need to filter out by workspace id, retrieving mentions first
-  // would lead to retrieve every single messages
-  const mentions = await Conversation.findAll({
+  const agentConfigurations = await AgentConfiguration.findAll({
+    attributes: ["sId"],
+    where: {
+      workspaceId: workspaceId,
```

Comment:
filter by status active

Reply:
important since otherwise it'll grab all versions so the list will be 10 to 100 times as big